### PR TITLE
Create a quiet option for driver.py

### PR DESCRIPTION
**Pull request by pmarkee**
JSRemoteTest-DCO-1.0-Signed-off-by: Peter Marki marpeter@inf.u-szeged.hu

### CODE REVIEW

**Comment by pmarkee** [comment]
Right now the testrunner generates huge outputs, which is not ideal. It would be useful if we had less verbose outputs by default, especially on Travis CI, that's what this patch does. The verbosity can still be turned on if you need it for debugging for example.

The reason there has to be separate JSVERBOSE and VERBOSE variables is because VERBOSE is checked by Jerryscript's ``build.py`` as well and it will display even more info, for example each gcc command that is executed, which is rarely needed. That's why we have JSVERBOSE, which makes js-remote-test more verbose, but not Jerryscript. I hope that makes sense.

File: API/common/utils.py [hunk 1]
```diff
@@ -41,7 +41,15 @@ def execute(cwd, cmd, args=None, quiet=False):
     stdout = None
     stderr = None
 
-    if quiet:
+    verbose = os.environ.get('VERBOSE', '')
+    jsverbose = os.environ.get('JSVERBOSE', '')
+
+    if verbose and not jsverbose:
+        console.warning('Please define JSVERBOSE environmental variable to enable verbose output')
```
Comment (robertsipka) [inline API/common/utils.py]: Ok, thanks, I understand now. This solution will print this message many times. You should move this check to `driver.py`.

**Comment by LaszloLango** [comment]
typo: `veriables` -> `variables`

**Comment by rtakacs** [comment]
`utils.patch` function uses `subprocess.call` instead of `utils.execute`. Could you please modify the `utils.patch` function to use the `utils.execute` implementation?

**Comment by pmarkee** [comment]
Thanks @robertsipka @LaszloLango @rtakacs , I updated the PR.

**Comment by rtakacs** [comment]
An other idea: what about having a `--quiet` option? If that was enabled, the `driver.py` would define a `QUIET` or `NO_VERBOSE` environment variable, that the `executor` can check (like now).

**Comment by pmarkee** [comment]
I updated the patch according to the suggestion by @rtakacs

**Pull request merged**